components/blitz/resources/omero/cmd/FS.ice

```diff
@@ -185,41 +185,6 @@ module omero {
 
         };
 
-        /**
-         * Request to determine the disk usage of the given objects
-         * and their contents. File-system paths used by multiple objects
-         * are de-duplicated in the total count. Specifying a class is
-         * equivalent to specifying all its instances as objects.
-         *
-         * Permissible classes include:
-         *   ExperimenterGroup, Experimenter, Project, Dataset,
-         *   Folder, Screen, Plate, Well, WellSample,
-         *   Image, Pixels, Annotation, Job, Fileset, OriginalFile.
-         **/
-        class DiskUsage extends Request {
```

Comment:
I have some small concern over the re-definition of these classes, i.e. giving `DiskUsage` a new parent class. It may be find in a major release, but this may be another case of bumping the class name to `DiskUsage2` since we've done that in this package for other classes.